sunpy/net/dataretriever/sources/tests/test_eve.py

```diff
@@ -134,29 +134,29 @@ def mock_query_object(LCClient):
     """
     # Creating a Query Response Object
     start = '2016/1/1'
-    end = '2016/1/2'
+    end = '2016/1/1 23:59:59'
```

Comment:
Just out of curiousity - why are all the end times for the tests (incl. ones for other instruments) all changed to 2016/01/01 23:59:59 rather than just 2016/01/02? - is this left over from the problem with scraper before?

Reply:
This is query response object to check the end time from the responses returned from table.
Earlier we defaulted it to end date for some clients and enddate - 1 microseconds for some. Now it is consistent to enddate -1 microseconds.